packages/fullScreen/web/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "wix-rich-content-fullScreen",
```

fullscreen is one word - please remove camelCase
```suggestion
  "name": "wix-rich-content-fullscreen",
```